writer/writer/core/writer_service.py

```diff
@@ -26,13 +26,16 @@ class WriterService:
 
     def write(self, write_request: WriteRequest) -> None:
         with self._lock:
+            print(f"Lock acquired")
```

Comment:
Can it be removed?

Reply:
Ahh, old debug statements... It's been a while since I worked on the datastore :D